subsys/usb/device_next/class/usbd_msc.c

```diff
@@ -112,13 +111,16 @@ struct msc_bot_ctx {
 	struct msc_bot_desc *const desc;
 	const struct usb_desc_header **const fs_desc;
 	const struct usb_desc_header **const hs_desc;
+	uint8_t *scsi_bufs[MSC_NUM_BUFFERS];
 	atomic_t bits;
 	enum msc_bot_state state;
+	uint8_t scsi_bufs_used;
+	uint8_t num_in_queued;
+	uint8_t num_out_queued;
```

Comment:
From the last commit, the bulk in and bulk out will not be queued at the same time. right? Then IMO, the `num_in_queued` and `num_out_queued` can be removed and only used `scsi_bufs_used`.